lib/config/validation.spec.ts

```diff
@@ -1213,6 +1213,58 @@ describe('config/validation', () => {
       expect(warnings).toHaveLength(1);
     });
 
+    it('does not error of use of `global:` presets in global `globalExtends`', async () => {
```

Comment:
```suggestion
    it('does not error on use of `global:` presets in `globalExtends`', async () => {
```

global-only option :)